integration/solana/token.spec.ts

```diff
@@ -0,0 +1,101 @@
+import { getOrCreateAssociatedTokenAccount, createMint, TOKEN_PROGRAM_ID } from '@solana/spl-token';
```

Comment:
How about asking someone from Solana who knows typescript to review this file?

Reply:
This is just a test file, I don't it matters enough to bother someone else about it.

Reply:
I think everyone in solana is a far too busy to bother about this. However, if we have typescript will be used by users, then a review by a type-script type person would be very helpful.